<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavegameWidget</class>
 <widget class="QWidget" name="SavegameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Savegame Widget</string>
  </property>
  <layout class="QHBoxLayout" name="hlSavegameContent">
   <item>
    <widget class="QLabel" name="labSavegameStr">
     <property name="text">
      <string>The Third Way (100%) - 00/00/00 00:00:00</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="hsSavegameWidget">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="cmdCopy">
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cmdDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
